### PR TITLE
Manolo tlc ciel d5 5_3 to 5_3c

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Ciel/Day 5/05_01_CIEL05_3C.txt [hunk 1]
```diff
@@ -437,18 +438,18 @@ It looks like I was relying on Arcueid more than I thought I was...
 [sha:48bacd44ee95f6792430c6c942b0005652de2a49]{
 -- Page 525, Offset 12227.
 -- <吸血鬼|ヴローヴ>を俺ひとりでどうにかしよう、という発想自体が間違っていると言うか、
-Or rather than just imagining I alone could somehow do something against Vlov is delusional.
+Just imagining I could somehow do something against Vlov alone is delusional.
 }
 [sha:981ba06a244bba582fbe79ac26cd3fa253e7dea2]{
 -- Page 525, Offset 12228.
 -- 反則には反則をぶつければいい、の精神と言うか……。
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_3C.txt:445]: this is more like "a 'foul play begets foul play' mindset" right?

File: script/Ciel/Day 5/05_01_CIEL05_3C.txt [hunk 2]
```diff
@@ -548,17 +549,18 @@ No matter what, I can't let the tragedy of what happened at the hotel repeat aga
 [sha:054c650ad6be9cdd0ef2598f9524d7e5edfb28e8]{
 -- Page 535, Offset 12249.
 -- 「―――分かった。頼むアルクェイド。
-"―――I understand. I ask of you Arcueid.
+// begging might be a bit too strong here
+"―――I understand. I'm begging you, Arcueid.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_3C.txt:553]: next line already starts with please so i say just drop it... as dangerous as that is with voiced lines
>I understand. Arcueid―――

**Review by Avuxo — APPROVED** [review]
otsukare, another rough set of files

File: script/Ciel/Day 5/05_01_CIEL05_3B.txt [hunk 3]
```diff
@@ -238,7 +239,8 @@ This isn't the time for idle talk!"
 [sha:83a692a1cf17fcd34cdb50fa329ac91f3c32268a]{
 -- Page 457, Offset 12073.
 -- 「率直に言います。一般人である貴方に、これ以上関与されては迷惑です。賢明な判断による黙秘を期待します。
-"Let me be frank. It would be inconvenient for you, a normal human, to be involved any further. I hope that you'll wisely decide to keep this matter under wraps.
+// "normal human" tohno shiki
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Ciel/Day 5/05_01_CIEL05_3B.txt:242]: on second thought that does make sense

i think "civilian" is right here?

File: script/Ciel/Day 5/05_01_CIEL05_3C.txt [hunk 4]
```diff
@@ -292,7 +293,7 @@ Senpai and Sensei are both good people, you know?! They've saved me so many time
 [sha:a7920d42595f052d599469558c0886035d542902]{
 -- Page 512, Offset 12198.
 -- わたしだって志貴を助けたのに！　頼りにされるのはわたしの方じゃないの!?」
-And I even help you too! Shouldn't you rather rely on me?!"
+%{i}I've%{/i} helped you too! Isn't it %{i}me%{/i} you should be relying on instead?!"
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Ciel/Day 5/05_01_CIEL05_3C.txt:296]: ciel my beloved

**Pull request merged**